HabitRPG/Views/ToastView.swift

```diff
@@ -121,6 +121,14 @@ class ToastView: UIView {
         loadOptions()
     }
     
+    public convenience init(goldDiff: Float, background: ToastColor, duration: Double? = nil, delay: Double? = nil) {
+        self.init(frame: CGRect.zero)
+        accessibilityLabel = "You received "
+        addStatsView(HabiticaIcons.imageOfGold, diff: goldDiff, label: "Gold")
```

Comment:
Can you make this and the Line above it translatable by using the L10n class that SwiftGen generates?

Reply:
Looking this over, is 

```
accessibilityLabel = "You received "
```
necessary? I believe I added this since the init above it had this var set.

Reply:
@phillipthelen bumping this up to my question above

Reply:
Sorry I forgot to respond. I believe this is needed, so that VoiceOver can read the toast